File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/collectionsheetindividualdetails/OnRetrieveSheetItemData.java
package com.mifos.mifosxdroid.online.collectionsheetindividualdetails;

import com.mifos.api.model.BulkRepaymentTransactions;

/**
 * Created by aksh on 20/6/18.
 */

public interface OnRetrieveSheetItemData {
    void onShowSheetMandatoryItem(BulkRepaymentTransactions transaction, int position);

    void onSaveAdditionalItem(BulkRepaymentTransactions transaction, int position);
}

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/collectionsheetindividualdetails/OnRetrieveSheetItemData.kt
package com.mifos.mifosxdroid.online.collectionsheetindividualdetails

import com.mifos.core.model.BulkRepaymentTransactions

/**
 * Created by aksh on 20/6/18.
 */
interface OnRetrieveSheetItemData {
    fun onShowSheetMandatoryItem(transaction: BulkRepaymentTransactions, position: Int)
    fun onSaveAdditionalItem(transaction: BulkRepaymentTransactions, position: Int)
}